homeassistant/components/xiaomi_miio/const.py

```diff
@@ -196,12 +196,12 @@ class SetupException(Exception):
     + MODELS_LIGHT_MONO
 )
 
-# TODO: use const from pythonmiio once new release with the constant has been published. # pylint: disable=fixme
```

Comment:
If this is no longer the plan, we should add a new comment at minimum about what to do with these constants.

Reply:
Fair enough, I added it back for the time being.